### PR TITLE
Feature/volume

### PR DESCRIPTION
**Pull request by willread**
## Description
Add volume support to the rise-video component.

## Motivation and Context
Users of the component need to be able to control the volume of videos and know that the volume will be muted by default.

Implemented according to [this section of the component design](https://docs.google.com/document/d/1LG_h_jQX8UlGLHfmZhEch0DCQyeVxmCnnDAUyU0CaoQ/edit#heading=h.wu8l3nc6gh0y)

## How Has This Been Tested?
- Both unit and integration tests have been added for new code
- Changes have been tested on both Electron and Chrome extension players using [this presentation](https://apps.risevision.com/templates/edit/3dfa38f2-7b70-4abe-858e-4cb14f81d139) which uses a test template with a sample video containing audio and a range slider which dynamically adjusts the volume of the video by updating the `volume` attribute of the `rise-video` component

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed.
  - Manual Test: Completed as noted above
  - Automated Test: Completed as noted above
  - Monitoring: Since it's Friday, we'll wait to merge this PR until Monday
  - Rollback: The previous release has been tagged so rollback should be simple
  - Documentation: The documentation has been updated with information about the new `volume` attribute
  - Internal Communication: Support does not need to be notified as the component is not in use 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No release checklist items were skipped.

### CODE REVIEW

File: src/rise-video-player.js [hunk 1]
```diff
@@ -228,6 +239,34 @@ export default class RiseVideoPlayer extends LoggerMixin( RiseElement ) {
...
+  _setVolume( volume ) {
+    if ( !this._playerInstance ) {
+      return;
+    }
+
+    const normalizedVolume = clampNumber( volume, 0, 100 );
+
+    this.volume = normalizedVolume;
+
+    // TODO: Check for valid display id
```
Comment (santiagonoguez) [inline src/rise-video-player.js]: @willread is this comment something related to the original code, can you clarify ?

File: src/rise-video.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ export default class RiseVideo extends WatchFilesMixin( ValidFilesMixin( RiseEle
           height: 100%;
         }
       </style>
-      <rise-video-player files="{{_filesToRenderList}}"></rise-video-player>
+      <rise-video-player id="videoPlayer" files="{{_filesToRenderList}}" volume="[[volume]]"></rise-video-player>
```
Comment (santiagonoguez) [inline src/rise-video.js]: @willread just to confirm, and id in shadow DOM never collides with other compoents', right ?

Comment (willread) [reply src/rise-video.js]: @santiagonoguez Nope, my understanding is that it should be encapsulated.

File: src/rise-video-player.js (hunk 1 above)
Comment (willread) [reply src/rise-video-player.js]: @santiagonoguez There was a check in the `widget-video` implementation against the following function and muted the video if it failed. I assume to mute the video when in preview mode. Since we have a later task to not play videos at all in preview mode maybe I can just remove this comment?

```  function isValidDisplayId() {
    return _displayId && _displayId !== "preview" && _displayId !== "display_id" && _displayId.indexOf( "displayId" ) === -1;
  }```

Comment (santiagonoguez) [reply src/rise-video-player.js]: @willread oh I see, that's to avoid volume during preview and test instances, I guess we can remove that check and comment for the time being. Thanks !
